stories/SQFormDropdown.stories.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+
+import {SQForm, SQFormDropdown as SQFormDropdownComponent} from '../src';
+
+export default {
+  title: 'Components/SQFormDropdown',
+  component: SQFormDropdownComponent,
+  parameters: {
+    docs: {
+      source: {
+        type: 'code'
```

Comment:
Added this as a workaround to fix a react error relating to storybook
https://github.com/storybookjs/storybook/issues/11543

Only downside is that in the "Docs" tab when click "Show code" we get the raw story rather than the component with props
https://storybook.js.org/docs/react/writing-docs/doc-blocks#docspage-1